src/v0/destinations/campaign_manager/util.test.js

```diff
@@ -0,0 +1,23 @@
+const { convertToMicroseconds } = require('./util');
+
+describe('convertToMicroseconds utility test', () => {
+  it('ISO 8601 input', () => {
+    expect(1609748704780000).toEqual(convertToMicroseconds('2021-01-04T08:25:04.780Z'));
```

Comment:
```suggestion
    expect(convertToMicroseconds('2021-01-04T08:25:04.780Z')).toEqual(1609748704780000);
```
Can we do this quick correction for all the test-cases in this file ?

Reply:
updated